test/system/030-run.bats

```diff
@@ -815,4 +815,11 @@ EOF
     run_podman run --uidmap 0:10001:10002 --rm --hostname ${HOST} $IMAGE grep ${HOST} /etc/hosts
     is "${lines[0]}" ".*${HOST}.*"
 }
+
+@test "podman run doesn't override oom-score-adj" {
+    current_oom_score_adj=$(cat /proc/self/oom_score_adj)
+    run_podman '?' run --rm $IMAGE cat /proc/self/oom_score_adj
```

Comment:
```suggestion
    run_podman run --rm $IMAGE cat /proc/self/oom_score_adj
```
We should make sure exit code is 0 or is there a special reason to not check for that?